test/assets/messages/CalculationMessages.scala

```diff
@@ -45,23 +45,41 @@ object CalculationMessages {
 
   val th4LPP = "Amount left to pay"
 
+  val th4LPPEstimated = "Left to pay"
+
   val th4Additional = "Calculation"
 
   val h2Additional = "Estimates"
 
-  val p2Additional = "Penalties and interest will show as estimates if HMRC does not have enough information to calculate the final amounts."
+  val p2AdditionalLPP2 = "Penalties and interest will show as estimates if HMRC does not have enough information to calculate the final amounts."
+
+  val p2AdditionalLPP1 = "Penalties and interest will show as estimates until:"
+
+  val p2EstimatedLPP1 = "The calculation we use is: 2% of £10,000.00 (Central assessment amount unpaid on 22 May 2025)"
```

Comment:
The content here doesn't look right, it should be The calculation we use is: 2% of £10,000.00 (the unpaid VAT 15 days after the due date)

Reply:
Just asked Linds to see if we need to say CA or VAT - will let you know, however the above still applies

Reply:
So, we should always refer to VAT therefore, this applies https://github.com/hmrc/penalties-frontend/pull/217#discussion_r925515717

Reply:
Please remove